carma-messenger-ui/website/scripts/index.js

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (C) 2018-2020 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+/***
+ This file shall contain ROS related function calls.
+****/
+
+// *** Global variables ***
+var retry_counter=0;
+var isROSConnected=false;
+
+$(document).ready(function () {       
+    $("#jqxLoader").jqxLoader({width: 150, height: 100, imagePosition: 'center', isModal: true});
+    $('#btnStart').on('click', function () {
+
+            //If launching backend platform, call the waitForROSConnection since starting backend service takes time;
+            if($('input[name=remotelaunch]').prop('checked')) {
```

Comment:
update "remotelaunch" to "launchplatform", see comment above. Applies to all with this naming convention.

Reply:
Addressed